04-database.md

```diff
@@ -82,7 +82,7 @@ But wait, there's more! If you dive into that `openConnection()` call, you're ta
 
 You may (or may not) remember that that global variable was created in the `DRUPAL_BOOTSTRAP_CONFIGURATION` phase of the bootstrap process, courtesy of the [`drupal_settings_initialize`](https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/drupal_settings_initialize/7) function.
 
-Anyways, the end result if that, back in the `Database::openConnection()` function, we have a fully populated `$databaseInfo` variable, and we pass that into the constructor for the driver class we're using. 
+Anyways, as a result of the `Database::openConnection()` function, we have a fully populated `$databaseInfo` variable, and we pass that into the constructor for the driver class we're using. 
```

Comment:
There's no way I'm going to pass that up if you feel up to it!

On Sun, Aug 9, 2015, 19:13 Robert <notifications@github.com> wrote:

> In 04-database.md
> <https://github.com/mikecrittenden/drupal-7-deconstructed/pull/10#discussion_r36595079>
> :
>
> > @@ -82,7 +82,7 @@ But wait, there's more! If you dive into that `openConnection()` call, you're ta
> >
> >  You may (or may not) remember that that global variable was created in the `DRUPAL_BOOTSTRAP_CONFIGURATION` phase of the bootstrap process, courtesy of the [`drupal_settings_initialize`](https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/drupal_settings_initialize/7) function.
> >
> > -Anyways, the end result if that, back in the `Database::openConnection()` function, we have a fully populated `$databaseInfo` variable, and we pass that into the constructor for the driver class we're using.
> > +Anyways, as a result of the `Database::openConnection()` function, we have a fully populated `$databaseInfo` variable, and we pass that into the constructor for the driver class we're using.
>
> If you think it would be helpful to bang out a sequence diagram for this
> process, I'd be glad to do that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mikecrittenden/drupal-7-deconstructed/pull/10/files#r36595079>
> .
>